pkg/bootstrap/bootstrap_config.go

```diff
@@ -416,7 +416,7 @@ func writeBootstrapForPlatform(config *meshconfig.ProxyConfig, node string, epoc
 	// Support multiple network interfaces
 	meta[model.NodeMetadataInstanceIPs] = strings.Join(nodeIPs, ",")
 
-	if opts["sds_uds_path"] != nil && opts["sds_token_path"] != nil {
+	if opts["sds_path"] != nil && opts["sds_token_path"] != nil {
```

Comment:
I think we should defer this renaming. We want to minimize the change to branch 1.3 :)

Reply:
Reverted this naming change.